domain/tutoringModel/model.py

```diff
@@ -29,15 +32,49 @@ def serialize(self):
             'calculated_on': self.calculated_on
         }
 
-    def get_learning_path(self, student_id, learning_style, algorithm):
-        if algorithm == "Graf":
+    def get_learning_path(self,
+                          student_id,
+                          learning_style,
+                          algorithm,
+                          list_of_les):
```

Comment:
Vllt lower case. Ist gute Fehlerprävention. (siehe Uper case in vorherigem commit)
```suggestion
    def get_learning_path(self,
                          student_id,
                          learning_style,
                          _algorithm,
                          list_of_les):
        algorithm = _algorithm.lower()
```

Reply:
Jup, ist zwar im Endpunkt schon drin, aber sollte vorsichtshalber überall gemacht werden